code/src/App.js

```diff
@@ -1,9 +1,87 @@
-import React from 'react';
+import React, { useState}  from 'react';
+import Header from 'Components/Header';
+import UserName from 'Components/UserName';
+import Transportation from 'Components/Transportation';
+import Active from 'Components/Active';
+import Temperature from 'Components/Temperature';
+import Summary from 'Components/Summary';
+
 
 export const App = () => {
+  const [counter, setCounter] = useState(1); //Counts questions
+  const [userName, setUserName] = useState(''); // 1 question UserName
+  const [transportation, setTransportation] = useState(''); // 2 question Transportation
+  const [active, setActive] = useState(''); // 3 question Active
+  const [temperature, setTemperature] = useState(''); // 4 question Temperature
+
+  const handleNextQuestionClick = () => {
+    setCounter(counter + 1);
+  };
+
+  const handlePreviousQuestionClick = () => {
+    setCounter(counter - 1);
+    };
+
+  const handleStartOverClick = () => {
+    location.reload()
+  };
+  
   return (
     <div>
-      Find me in src/app.js!
-    </div>
+      <main>
+<Header/>
```

Comment:
Really great job with your App.js! The code is very clean and you basically almost only import your components.